tests/rptest/services/redpanda_installer.py

```diff
@@ -341,6 +343,13 @@ def highest_from_prior_feature_version(self, version):
         )
         return result
 
+    def latest_for_line(self, version: Tuple[int, int]):
+        """
+        Returns the most recent version of redpanda from a release line.
+        e.g: latest_for_line((22.2)) -> (22.2.7)
+        """
+        return self.highest_from_prior_feature_version((version[0], version[1]+1))
```

Comment:
Just a thought: maybe a less intrusive change would be to allow `install()` to check whether `version` is a 2-sized tuple and call `latest_for_line()` under the hood

Reply:
I like this idea